packages/plugin-react-native-navigation/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@bugsnag/plugin-react-native-navigation",
+  "version": "7.3.5",
+  "main": "react-native-navigation.js",
+  "description": "@bugsnag/react-native plugin for integration with react-native-navigation",
+  "homepage": "https://www.bugsnag.com/",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:bugsnag/bugsnag-js.git"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "*.js"
+  ],
+  "author": "Bugsnag",
+  "license": "MIT",
+  "devDependencies": {
+    "@bugsnag/core": "^7.3.5"
+  },
+  "peerDependencies": {
+    "@bugsnag/core": "^7.0.0",
+    "react-native-navigation": "^7.0.0"
```

Comment:
Is this incompatible with older versions? Could this version constraint be more loose?

Reply:
Widened it down to v2 as the same API has existed since then